### PR TITLE
feat: add AWS_PINPOINT_DEFAULT_POOL_ID to k8s

### PR DESCRIPTION
**Pull request by sastels**
## What happens when your PR merges?
- Prefix the title of your PR:
    - `feat:` - tag `main` as a new minor release

## What are you changing?
- [ ] Releasing a new version of Notify
- [x] Changing kubernetes configuration

## Provide some background on the changes
Add the new `AWS_PINPOINT_DEFAULT_POOL_ID` env var to k8s (celery sms sending pods). Initially set the value to empty on both staging and production.
```
$ make diff-staging
> AWS_PINPOINT_DEFAULT_POOL_ID=
```

```
$ make diff-production
> AWS_PINPOINT_DEFAULT_POOL_ID=
```

We had a few issues making these changes before but I believe the problems have been sorted out. Local testing with
```
AWS_PINPOINT_DEFAULT_POOL_ID=
```
works, and this PR uses an empty `AWS_PINPOINT_DEFAULT_POOL_ID` for both staging and production, so we should be able to verify this configuration on staging before deploying it to production.

## Checklist if making changes to Kubernetes:
- [x] I know how to get kubectl credentials in case it catches on fire

## After merging this PR
- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

File: env/staging/.env.zip.enc.aws
Comment (jimleroyer) [inline env/staging/.env.zip.enc.aws:1]: So ideally, this is where this information would rather be contained in the database, leveraging what the app team will do, right?

**Comment by jimleroyer** [comment]
@sastels The encrypted values are empty at the moment for both staging and production, from this PR?

File: env/staging/.env.zip.enc.aws
Comment (sastels) [reply env/staging/.env.zip.enc.aws:1]: Right now I just added a template list to these environment variables, but in the future we could decide more dynamically what to send with the shortcode (ie going by template priority, for example)

Comment (jimleroyer) [reply env/staging/.env.zip.enc.aws:1]: I just saw an empty variable when I checked for both of these locally. Is that normal?

Comment (sastels) [reply env/staging/.env.zip.enc.aws:1]: The `AWS_PINPOINT_DEFAULT_POOL_ID` is empty on staging and prod, but the `AWS_PINPOINT_SC_POOL_ID` and `AWS_PINPOINT_SC_TEMPLATE_IDS` variables have values on both. So currently we use Pinpoint for the `AWS_PINPOINT_SC_TEMPLATE_IDS` template but SNS for the rest. This won't change with this PR, but next we'll set the `AWS_PINPOINT_DEFAULT_POOL_ID` variable and use the Pinpoint default pool for the rest.

Comment (jimleroyer) [reply env/staging/.env.zip.enc.aws:1]: I looked too quickly and did not see these, you are right and these have values!

**Review by jimleroyer — APPROVED** [review]

**Pull request merged**